### PR TITLE
WT-6690 Add support for table import when the exported configuration is provided

### PR DESCRIPTION
**Pull request by tetsuo-cpp**
Support for table imports isn't materially different from importing files.
This PR is just a Python test to explicitly check that this works and exercise some table specific functionality like projections.

### CODE REVIEW

File: test/suite/test_import03.py [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+        # Read the entire record.
+        cursor = self.session.open_cursor(uri)
+        self.session.begin_transaction('read_timestamp=' + timestamp_str(read_ts))
+        cursor.set_key(key)
+        self.assertEqual(0, cursor.search())
+        self.assertEqual(list(value_tuple), cursor.get_value())
+        self.session.rollback_transaction()
+        cursor.close()
+
+        # Now let's test something table specific like a projection.
```
Comment (tammybailey) [inline test/suite/test_import03.py:70]: We only need to support simple tables for this project.

Comment (tetsuo-cpp) [reply test/suite/test_import03.py:70]: It tests that the table specific configuration (not `file_metadata`) is getting used properly.
It works so I don't see a reason to remove this.

File: src/schema/schema_create.c [hunk 2]
```diff
@@ -604,10 +605,15 @@ __create_table(WT_SESSION_IMPL *session, const char *uri, bool exclusive, const
 
     tablename = uri;
     WT_PREFIX_SKIP_REQUIRED(session, tablename, "table:");
+    import = __wt_config_getones(session, config, "import.enabled", &cval) == 0 && cval.val != 0;
 
     /* Check if the table already exists. */
     if ((ret = __wt_metadata_search(session, uri, &tableconf)) != WT_NOTFOUND) {
-        if (exclusive)
+        /*
+         * Don't allow a file to get imported if it already exists in our database. This is most
```
Comment (tetsuo-cpp) [inline src/schema/schema_create.c:613]: Repeat 9f8ec12 here too.

**Comment by tammybailey** [comment]
@tetsuo-cpp would you mind taking a look at my changes?

**Comment by tetsuo-cpp** [comment]
> @tetsuo-cpp would you mind taking a look at my changes?

I can't LGTM since it's my own PR but the changes look good.

**Review by tammybailey — APPROVED** [review]
LGTM

**Review by sulabhM — APPROVED** [review]

**Pull request merged**